lua/smart-splits/mux/tmux.lua

```diff
@@ -28,7 +28,12 @@ local function tmux_exec(args, as_list)
     return nil
   end
 
-  local cmd = vim.list_extend({ 'tmux', '-S', socket }, args, 1, #args)
+  local cmd = nil
+  if os.getenv('TMUX_PANE') then
+    cmd = vim.list_extend({'flatpak-spawn', '--host', 'tmux', '-S', socket }, args, 1, #args)
```

Comment:
don't we need to check if we're actually in a flatpak first? `TMUX_PANE` can be set without being in a flatpak.

Reply:
Absolutely. I was hurring too much with the commit and eded up commiting a test version. Now fixed.